content/python/concepts/json-module/terms/json-load/json-load.md

```diff
@@ -0,0 +1,55 @@
+---
```

Comment:
Please change the folder name from `json-load`--> `load` and file name from `json-load.md` --> `load.md`